.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Documentation 📝
+about: Suggest better docs coverage for a particular tool or process.
+labels: "type: documentation"
```

Comment:
```suggestion
labels: "docs"
```

Reply:
yea I didn't want it to be the same as the one we use for "this module has docs" but if you think they should be the same I can do that for sure

Reply:
Hm, I guess `docs` as it exists at the minute is actually `has-docs` or something else. I think the existing use of flags is arbitrary, other than the `rn-*` flags which serve a purpose in release notes automation.

I've no strong preference here,  potentially we could go with:
- `docs-request`
- `module-request`
- `feature-request`

Or am I happy to stick with `docs` for issues and pull requests for simplicity/consistency.

Reply:
I think the request prefix is a good shout